src/index/base.cpp

```diff
@@ -95,17 +95,18 @@ void BaseIndex::ThreadSync()
         int64_t last_locator_write_time = 0;
         while (true) {
             if (m_interrupt) {
-                WriteBestBlock(pindex);
+                m_best_block_index = pindex;
```

Comment:
Previously, since `m_synced` wasn't set to *and* WriteBestBlock took an explicit pointer argument, it wasn't necessary. But now WriteBestBlock reads m_best_block_index instead of taking an argument.